### PR TITLE
Feature liveplayer

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -177,9 +178,142 @@ Site.Support = {
...
+    });
+
+    _this.playerIsInitialized = true;
+  },
+
+  isSchedule: function() {
+    var _this = this;
+
+    var utc = new Date();
+    var now;
```
Comment (mixcas) [inline js/main.js:262]: What's the point to declare `now` here?

File: js/main.js [hunk 2]
```diff
@@ -177,9 +178,142 @@ Site.Support = {
...
+
+    if (_this.isBST) {
+      now = new Date(utc.getUTCFullYear(), utc.getUTCMonth(), utc.getUTCDate(), (utc.getUTCHours() + 1), utc.getUTCMinutes(), utc.getUTCSeconds());
```
Comment (mixcas) [inline js/main.js:265]: What about something like

```
utc.setHours(utc.getHours() + 1);
now = utc;
```

`now` might not be needed.

Comment (varffs) [inline js/main.js:265]: tbh this is just migrated code from the existing codebase. If you want to rewrite it then I'm open to it but it isnt as simple as just adding an hour [as adding an hour might make it the next day] I dont think

Comment (mixcas) [inline js/main.js:265]: i see. the code is pretty hard to read. i couldn't rewrite it cuz i don't fully understand the schedule. 

what do? rise an issue? add a comment?

Comment (varffs) [inline js/main.js:265]: whats the issue? the code needs commenting to be clear?

**Comment by mixcas** [comment]
@varffs `isSchedule()` and `isBST()` are hard to read and understand, I don't know the context or what some of those vars mean. I know they are related to a specific schedule and something about dates. The date handling is kinda messy. Do you think this something that could be improved in the future? I hard for me to tell since i don't know the full spec of how it should work. 

If it could be improved we leave it as is and rise an issue for it. If it's fine like that maybe some comments on the code to help understand the basics of whats going on.

**Comment by varffs** [comment]
I'll add some commenting to the js file then merge once you get it yh?

**Comment by mixcas** [comment]
@varffs yeah, besides that everything seems fine

**Comment by varffs** [comment]
@mixcas there you go

**Pull request merged**